### PR TITLE
Removing option to fix compare notice & adding padding under it

### PR DESCRIPTION
**Pull request by ScopeyNZ**
Removed the option to "fix" the notice because it's always fixed anyway 😅

### CODE REVIEW

File: client/src/components/HistoryViewer/HistoryViewer.js [hunk 1]
```diff
@@ -65,6 +66,22 @@ class HistoryViewer extends Component {
...
+  /**
+   * Returns a string to be used as the "class" attribute on the history viewer container
+   *
+   * @returns {string}
+   */
+  getContainerClasses() {
+    const classes = ['history-viewer', 'fill-height'];
+
+    if (this.props.compare) {
+      classes.push('compare-mode');
```
Comment (robbieaverill) [inline client/src/components/HistoryViewer/HistoryViewer.js]: can you make this `history-viewer--compare-mode` please?

Comment (ScopeyNZ) [reply client/src/components/HistoryViewer/HistoryViewer.js]: Done 🙂

Comment (ScopeyNZ) [reply client/src/components/HistoryViewer/HistoryViewer.js]: Wait, it should be `history-viewer__compare-mode` - one moment please

Comment (ScopeyNZ) [reply client/src/components/HistoryViewer/HistoryViewer.js]: Ok actually done 🙂

**Pull request merged**